harness/determined/tensorboard/build.py

```diff
@@ -78,6 +78,7 @@ def build(
     checkpoint_config: Union[Dict[str, Any], str],
     container_path: Optional[str] = None,
     async_upload: bool = True,
+    sync_on_close: bool = True,
```

Comment:
This isn't actually used in anything here. Pass `sync_on_close` to the shared_fs/gcs/s3/azure classes?

Reply:
fixed